wl-js-analytics-adapter.sublime-snippet

```diff
@@ -20,14 +20,9 @@ WEBLINC.analytics.registerAdapter('${1:adapterName}', function () {
             var scripts = _.toArray(document.scripts),
 
             loaded = _.any(scripts, function (script) {
-                return script.src.match(WEBLINC.config.${1:adapterName}.scriptRegex);
+                return script.src.match(/{3:someregexthatmatchesyour3rdpartyscript}/i);
```

Comment:
You didn't increment the numbers that follow. Line 38 (for example) also is a "3".